src/controllers/order.controllers.js

```diff
@@ -0,0 +1,24 @@
+const postOrderService = require('../services/order.service');
+
+const postOrder = async (req,res) => {
+    try{
+        
+        const { user_id, order_number, status_id,  price, quantity } = req.body;
```

Comment:
```suggestion
        const { order_number, status_id,  price, quantity } = req.body;
```
order 관련 API는 인가과정이 필요한 API입니다. user_id 는 프론트엔드에서 보내주는 body로 확인하는 것이 아닌, jwt를 미들웨어로 검증하는 과정에서 갖고 오면 좋을 것 같습니다. 이는 진주님 코드에서 제가 가이드해드린 내용이 있으니 여쭤보고 같이 확인해보시면 될 것 같아요!